worker/Makefile

```diff
@@ -74,7 +75,7 @@ ifeq ($(wildcard $(PIP_DIR)),)
 		echo "Installation failed, likely because PIP is unavailable, if you are on Debian/Ubuntu or derivative please install the python3-pip package"
 	# Install `meson` and `ninja` using `pip` into custom location, so we don't
 	# depend on system-wide installation.
-	$(PYTHON) -m pip install --upgrade --target=$(PIP_DIR) $(PIP_BUILD_BINARIES) meson ninja
+	$(PYTHON) -m pip install --upgrade --target=$(PIP_DIR) $(PIP_BUILD_BINARIES) meson==$(MESON_VERSION) ninja
```

Comment:
Locking to an exact version means that pip cannot cleverly figure out which ones are compatible with your current system. A far, far, far better idea:

```
pip install "meson>=0.58,<=0.62.*"
```

This pins an allowable minimum/maximum range, which supports reasonably new Meson on systems with modern Python, while also handling archaic systems such as Ubuntu 18.04.

That being said, I'm probably duty-bound to say I'm not a fan of pinning versions... you'll need to babysit this to make sure it gets upgraded regularly, and I don't think tools such as dependabot or renovatebot (https://mend.io) can detect version pins as arbitrary lines in a Makefile.